content/FLASHTnT/FLASHTnTWorkflow.py

```
@@ -78,61 +78,38 @@ def process_uploaded_files(uploaded_files):
-
-                    # opening file dialog and closing without choosing a file results in None upload
-                    process_uploaded_files(uploaded_file)
-                    st.success("Successfully added uploaded files!")
-                else:
-                    st.warning("Upload some files before adding them.")
+        _, c2, _ = st.columns(3)
+        # User needs to click button to upload selected files
+        if c2.form_submit_button("Add files to workspace", type="primary"):
+            if uploaded_file:
+                # A list of files is required, since online allows only single upload, create a list
+                if type(uploaded_file) != list:
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)